lib/sqitchtutorial-mysql.pod

```diff
@@ -279,13 +281,13 @@ from the registry database:
   # Name:     appuser
   # Deployed: 2013-12-31 13:13:17 -0800
   # By:       Marge N. O’Vera <marge@example.com>
-  # 
```

Comment:
Much as I, too, am a stickler for eliminating trailing whitespace, I also tend to preserve it where it's output by the documented tools.

Reply:
Ah, sorry.. Must've made an edit to a file I didn't intend, and then I did the good old `git commit -a`; I'll fix!

Reply:
There are a bunch of those, esp. in the tutorial files.